Story 3:

SamuelConnor opened the pull request:
-CRUD functionality for Customer

SamuelConnor committed:
Story 3:
-CRUD functionality for Customer

Kieransullivan-imanage commented on src/main/java/com/example/demo/controllers/CustomerController.java:
```diff
@@ -0,0 +1,62 @@
+    public void deleteLocation(@PathVariable Long id)
+    {
+        if(customerService.findCustomerById(id) == null)
+            throw new LocationNotFoundException();
```
CustomerNotFoundException?

SamuelConnor committed:
Story 4:
-CRUD functionality for Car
-Endpoints added for:
-Location without cars
-Location with cars
-All locations without cars

SamuelConnor committed:
Story 3:
-Fixed wrong exception used.
